storio-common-annotations-processor/src/main/java/com/pushtorefresh/storio/common/annotations/processor/introspection/StorIOColumnMeta.java

```diff
@@ -41,6 +41,16 @@ public boolean isMethod() {
         return element.getKind() == ElementKind.METHOD;
     }
 
+    public String getRealElementName() {
+        if (elementName.startsWith("get") && Character.isUpperCase(elementName.charAt(3))) {
+            return elementName.substring(3).toLowerCase();
```

Comment:
You are right but the thing is that this feature is intended to use only for AutoValue and Kotlin cases. AutoValue classes don't need getters and Kotlin properties by default use `getProperty` format for getters. For all other cases, the right choice is just to annotate fields as it was supposed in a first place. Maybe we should emphasise it more in documentation.